spec/function.dd

```diff
@@ -1349,9 +1349,41 @@ void foo(int x, int y = 3)
 foo(4);   // same as foo(4, 3);
 ---
 
-        $(P Default parameters are evaluated in the context of the
+        $(P Default parameters are resolved and semantically checked in the context of the
         function declaration.
-        If the default value for a parameter is given, all following
+---
```

Comment:
Close the paragraph before starting a code section. HTML doesn't like `<pre>` in `<p>`.

Reply:
ok